hugegraph-dist/release-docs/NOTICE

```diff
@@ -28,6 +28,26 @@ limitations under the License.
 
 ========================================================================
 
+snowflake NOTICE
+
+========================================================================
+This product contains code form the snowflake Project:
+
+snowflake
+Copyright 2010-2012 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+file except in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+========================================================================
+
```

Comment:
where is the `NOTICE` file? seems we don't need license info here? @simon824 

We should check other NOTICE import to avoid `LICENSE` info in it

Reply:
https://github.com/twitter-archive/snowflake/blob/snowflake-2010/LICENSE

Reply:
we can't use license info in notice file, we need find notice file in their repo instead